.github/workflows/release.yml

```diff
@@ -14,6 +14,17 @@ jobs:
     runs-on: ubuntu-latest
     needs:
       - build
+    env:
```

Comment:
@AsifNawaz-cnic why have you changed the installation procedure of python dependencies in that way? With pip install --upgrade -e . it is automatically referencing to the setup.py.
Dependabot won't be able to automatically update those package lists in our workflows and this would mean additional manual work.

Reply:
I think I misunderstood your comment on the Jira issue "Find how to install the python deps on ci workflow". I will revert my changes however sphinx is working fine with the latest version as well.

Reply:
My thinking was that you can identify how to install the deps locally in our ci workflows to give that new sphinx version a try before we merge. I did not had much time while writing and the steps exactly in mind. Otherwise good to know that the new version is fully compatible. :-)